### DISCUSSION TITLE
Use in Unity Project

### DISCUSSION THREAD

**Discussion by warpdriv** [devlooped/GitInfo#325] [General]
Hi, 

Thank you for creating this package! I am trying to use this package in a Unity project. However, after I install it, I am unable to get a reference to it. Do you know how to resolve this?

Thank you for your help.

**Comment by kzu** [comment]
What do you mean by "get a reference to it"? I'm not familiar with how a Unity project builds, but if you can get an MSBuild output from it, you should see the GitInfo targets being invoked.
